locale/zh-tw/site.json

```diff
@@ -4,8 +4,8 @@
   "author": "Node.js",
   "url": "https://nodejs.org/zh-tw/",
   "locale": "zh-tw",
-  "language": "中文 (繁體)",
-  "languageEnglishVersion": "Traditional Chinese",
+  "language": "繁體中文",
+  "languageEnglishVersion": "Chinese Traditional",
```

Comment:
https://en.wikipedia.org/wiki/Traditional_Chinese_characters

`Traditional Chinese`

<img width="1239" alt="Screen Shot 2021-03-29 at 10 29 43 AM" src="https://user-images.githubusercontent.com/6274807/112779751-00dfaa00-907a-11eb-9de7-d2316658ff19.png">

Reply:
but in our english classes, we never use like this